external/payara-mp-tck/README.md

```diff
@@ -0,0 +1,36 @@
+<!--
```

Comment:
Not sure if this license paragraph is necessary or not @smlambert could you confirm that?

Reply:
Forgot to ask for it in the PRs, but it is okay to have it.